docs/deploy/howto-deploy-helm.md

```diff
@@ -27,6 +29,7 @@ Kubernetes cluster:
 - For `0.3.1` version of the Chart: [Azure Industrial IoT Helm Chart v0.3.1](https://github.com/Azure/Industrial-IoT/blob/helm_0.3.1/deploy/helm/azure-industrial-iot/README.md)
 - For `0.3.2` version of the Chart: [Azure Industrial IoT Helm Chart v0.3.2](https://github.com/Azure/Industrial-IoT/blob/helm_0.3.2/deploy/helm/azure-industrial-iot/README.md)
 - For `0.4.0` version of the Chart: [Azure Industrial IoT Helm Chart v0.4.0](https://github.com/Azure/Industrial-IoT/blob/helm_0.4.0/deploy/helm/azure-industrial-iot/README.md)
+- For `0.4.1` version of the Chart: [Azure Industrial IoT Helm Chart v0.4.0](https://github.com/Azure/Industrial-IoT/blob/helm_0.4.1/deploy/helm/azure-industrial-iot/README.md)
```

Comment:
Should this be 0.4.1 in the Chart name in the link?

Reply:
Good catch. Fixed.